CHANGES.md

```diff
@@ -1,5 +1,40 @@
+  - The top navigation has been updated to improve space allocation, with main nav links moved to the left.
+  - “Overview” is the new Dagit home page and “factory floor” view, were you can find the run timeline, which now offers time-based pagination. The Overview section also contains pages with all of your jobs, schedules, sensors, and backfills. You can filter objects by name, and collapse or expand repository sections.
+  - “Workspace” has been redesigned to offer a better summary of your repositories, and to use the same performant table views, querying, and filtering as in the Overview pages.
+- `@asset` and `@multi_asset` now accept a `retry_policy` argument. (Thanks Adam Bloom!)
+- When loading an input that depends on multiple partitions of an upstream asset, the `fs_io_manager` will now return a dictionary that maps partition keys to the stored values for those partitions. (Thanks andrewgryan!).
+- `JobDefinition.execute_in_process` now accepts a `run_config` argument even when the job is partitioned. If supplied, the run config will be used instead of any config provided by the job’s `PartitionedConfig`.
+- The `run_request_for_partition` method on jobs now accepts a `run_config` argument. If supplied, the run config will be used instead of any config provided by the job’s `PartitionedConfig`.
+- The new `NotebookMetadataValue` can be used to report the location of executed jupyter notebooks, and Dagit will be able to render the notebook.
+- Resolving asset dependencies within a group now works with multi-assets, as long as all the assets within the multi-asset are in the same group. (Thanks `@peay`!)
+- UPathIOManager, a filesystem-agnostic IOManager base class has been added - (Thanks `@****[danielgafni](https://github.com/danielgafni)`!)\*\*
+- A threadpool option has been added for the scheduler daemon. This can be enabled via your `dagster.yaml` file; check out the [docs](https://docs.dagster.io/deployment/dagster-instance#schedule-evaluation).
+- The default LocalComputeLogManager will capture compute logs by process instead of by step. This means that for the `in_process` executor, where all steps are executed in the same process, the captured compute logs for all steps in a run will be captured in the same file.
+- [dagster-airflow] `make_dagster_job_from_airflow_dag` now supports airflow 2, there is also a new mock_xcom parameter that will mock all calls to made by operators to xcom.
+- [helm] volume and volumeMount sections have been added for the dagit and daemon sections of the helm chart.
+
+### Bugfixes
+
+- For partitioned asset jobs whose config is a hardcoded dictionary (rather than a `PartitionedConfig`, previously `run_request_for_partition` would produce a run with no config. Now, the run has the hardcoded dictionary as its config.
```

Comment:
```suggestion
- For partitioned asset jobs whose config is a hardcoded dictionary (rather than a `PartitionedConfig`), previously `run_request_for_partition` would produce a run with no config. Now, the run has the hardcoded dictionary as its config.
```